.circleci/config.yml

```diff
@@ -384,37 +375,22 @@ workflows:
               only:
                 - development
     jobs:
-      - build
-      - coding_standards:
-          requires:
-            - build
-      - deprecated_code:
-          requires:
-            - build
-      - unit_kernel_tests:
-          requires:
-            - build
-      - edge_build:
-          requires:
-            - build
+      - edge_build
 
-  # A separate scheduled workflow to sync the data after the nightly build completes.
+  # A separate scheduled workflow to sync the data after the edge build completes.
   # This is separate to avoid using build time/compute units with arbitrary sleep commands;
   # platform.sh deploy hooks can take a few minutes to execute after commits are relayed to it.
-  nightly-content-sync:
+  edge-build-post-build-tasks:
```

Comment:
Renamed because it does a bit more than just sync content. It's basically a workflow to happen after the edge build has run. Would be nice if we could make it dependent on the success of another workflow but Circle CI doesn't stretch to that level... just dependencies between jobs in the same workflow.